packages/desktop-client/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -398,7 +398,10 @@ function SingleAutocomplete({
               },
               onBlur: e => {
                 // Should this be e.nativeEvent
-                e.preventDownshiftDefault = true;
+                // preventDownshiftDefault is a downshift.js property - they don't have types for it on the React event
+                // Suggested way to make this work with typescript is to cast as any... https://github.com/downshift-js/downshift/issues/734
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                (e as any).preventDownshiftDefault = true;
```

Comment:
@MatissJanis Fixing the Input.tsx types gave us an issue on this Autocomplete component by downshift.

The issue was that "preventDownshiftDefault" is not on the React.FocusEvent<HTMLInputElement, Element> - and rightly so 🤪 .

It's a known issue with downshift's types, here's the github issue: https://github.com/downshift-js/downshift/issues/734

The suggested workaround is to type it as "any" so that's what I've done here.

Reply:
Thanks! Very informative comment :)

Reply:
I think I just found a better way than (e as any). The types don't complain when I access it directly, like 

```
e['preventDownshiftDefault'] = true;
```

I've seen that elsewhere in the autocomplete file, so I've updated with that.